### DISCUSSION TITLE
attach strategies file to the jaeger-all-in-one docker run command option to run locally on mac

### DISCUSSION THREAD

**Discussion by AshokVardhanM** [jaegertracing/jaeger#4604] [Q&A] [answered]
How do I attach strategies yaml/json file to the jaeger-all-in-one docker run command to run locally on the mac.

I have tried with the following command but it always consider default values.
docker run -d   -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 -e COLLECTOR_OTLP_ENABLED=true     -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 4317:4317 -p 4318:4318 -p 14250:14250 -p 14268:14268 -p 14269:14269 -p 9411:9411  -e SAMPLING_CONFIG_FILE=/Users/ashokmallenipalli/Desktop/strategies.json -e SAMPLING_CONFIG_TYPE=file jaegertracing/all-in-one:latest

can someone please help in getting this addressed.
Thanks

**Comment by AshokVardhanM** [comment]
I also tried the following comamnd:
docker run -d   -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 -e COLLECTOR_OTLP_ENABLED=true     -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 4317:4317 -p 4318:4318 -p 14250:14250 -p 14268:14268 -p 14269:14269 -p 9411:9411   jaegertracing/all-in-one:latest --sampling.strategies-file="/Users/ashokmallenipalli/Desktop/strategies.json"

**Comment by yurishkuro** [comment, marked as answer]
Containers do not have access to your host file system by default, you need to mount a volume to map a host directory to some path within the container file system, and then refer to the file path within that container file system
```
docker run -v <host_directory>:<container_directory> <image>
```

**Reply by AshokVardhanM** [reply]
Yes I missed mounting and it worked, Thanks.
